tasks.py

```diff
@@ -24,7 +24,7 @@ def dbname(version):
 
 
 def assert_version(version):
-    assert version in ("15.0", "14.0", "13.0", "12.0", "11.0", "10.0", "9.0")
+    assert version in ("16.0", "15.0", "14.0", "13.0", "12.0", "11.0", "10.0", "9.0")
```

Comment:
do we want to remove 10 and 9 ?

Reply:
go point, I remove it as we do not build it.